src/CosmosDB/CosmosDB/Models/ManagedCassandra/PSManagedCassandraClusterGetPropertiesResource.cs

```diff
@@ -0,0 +1,132 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.CosmosDB.Models;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.Azure.Commands.CosmosDB.Models
+{
+    public class PSManagedCassandraClusterGetPropertiesResource
+    {
+        PSManagedCassandraClusterGetPropertiesResource()
+        {
+        }
+
+        public PSManagedCassandraClusterGetPropertiesResource(ClusterResourceProperties clusterProperties)
```

Comment:
>PSManagedCassandraClusterGetPropertiesResource [](http://example.com/codeflow?start=15&length=46)

nit: convention we follow is to append "PS" in the model name defined in SDK. Lets maintain that for consistency purposes.